build/testdata/024.golden

```diff
@@ -4,8 +4,8 @@
 str.rewrite(".", "/")
 
 str.rewrite(
-    before = ".",
     after = "/",
+    before = ".",
```

Comment:
Unfortunately that breaks a bunch of internal tests, so I don't think it can be included at this time.